core/api/src/domain/users/index.ts

```diff
@@ -39,6 +39,8 @@ export const checkedToEmailAddress = (
   return emailAddress as EmailAddress
 }
 
+export const DefaultLanguage = "" as UserLanguageOrEmpty
```

Comment:
With the default being set the type `UserLanguageOrEmpty` becomes irrelevant. Perhaps you could find something semantically more meaningful. Or if it's too complicated to untangle we can leave it for another PR.

Looks good otherwise.

Reply:
Not sure I fully understand. The `DefaultLanguage` is used as a fallback in the case of the error, but even in the case there is not an error the notifications service could still return an empty string implying that the mobile application should use the OS system default. It's definitely a little bit of a mess that is a relic of how the mobile application was handling the language.

Reply:
I think it may make sense to save that refactoring for a separate pr.